src/proteus/atmos_clim/janus.py

```diff
@@ -101,7 +101,7 @@ def UpdateStateAtm(atm, hf_row:dict, trppT:int):
             Dictionary containing simulation variables for current iteration
     """
 
-    atm.setSurfaceTemperature(hf_row["T_surf"])
+    atm.setSurfaceTemperature(hf_row["T_magma"])
```

Comment:
I never got the distinction between T_surf and T_magma but the surface temperature returned by the interior models is indeed T_magma.